src/tpl/faq.html

```diff
@@ -1,252 +1,310 @@
+          <div id="collapseDiv5" class="collapse-div" role="tablist">
+            <div class="collapse-header" id="heading5">
+              <button data-toggle="collapse" data-target="#collapse5" aria-expanded="false" aria-controls="collapse5">
+                5. Chi è il referente da indicare nel form?
+              </button>
+            </div>
+            <div id="collapse5" class="collapse" role="tabpanel" aria-labelledby="heading5">
+              <div class="collapse-body">
+                Il referente è la persona che effettua la richiesta di onboarding. Non è necessario che tale persona sia
+                il rappresentante legale dell’ente, ma è ovviamente necessario che abbia i permessi o le deleghe interne
+                per poter effettuare la richiesta. Tale nominativo sarà citato nella PEC indirizzata all’Amministrazione
+                richiedente in modo da facilitare lo smistamento del messaggio all’interno dell’Amministrazione stessa.
+                E’ necessario inserire nome e cognome del referente unitamente ad un numero di telefono che potrà essere
+                utilizzato per richiedere eventuali chiarimenti.
+              </div>
+            </div>
+          </div>
+          <div id="collapseDiv6" class="collapse-div" role="tablist">
+            <div class="collapse-header" id="heading6">
+              <button data-toggle="collapse" data-target="#collapse6" aria-expanded="false" aria-controls="collapse6">
+                6. Perché non posso scegliere l’indirizzo PEC di destinazione o indicare un altro indirizzo e-mail?
+              </button>
+            </div>
+            <div id="collapse6" class="collapse" role="tabpanel" aria-labelledby="heading6">
+              <div class="collapse-body">
+                L’indirizzo PEC è usato per confermare che il richiedente sia effettivamente l’Ente che
+                per cui si sta inserendo lo strumento di code hosting.
+
+                A quell’indirizzo arriverà una richiesta di conferma ed è quindi quello registrato in IndicePA
+                per quell’Ente.
+              </div>
+            </div>
+          </div>
+          <div id="collapseDiv7" class="collapse-div" role="tablist">
+            <div class="collapse-header" id="heading7">
+              <button data-toggle="collapse" data-target="#collapse7" aria-expanded="false" aria-controls="collapse7">
+                7. I dati riportati automaticamente non sono corretti. Come posso cambiarli?
+              </button>
+            </div>
+            <div id="collapse7" class="collapse" role="tabpanel" aria-labelledby="heading7">
+              <div class="collapse-body">
+                Se i dati riportati nei campi compilati automaticamente non dovessero essere corretti o non aggiornati è
+                necessario aggiornarli presso IndicePA. Per farlo si veda la <a
+                  href="https://www.indicepa.gov.it/documentale/n-domande.php#A3">sezione domande di IndicePA</a>.
```

Comment:
Broken link

Reply:
Probably this should work https://www.indicepa.gov.it/ipa-files/help-portale/index.html#come-viene-aggiornato-l-IPA but I'm not 100% sure though

Reply:
@libremente oops, I didn't notice your comment :) I used https://indicepa.gov.it/ipa-files/help-portale/index.html#servizi-enti